GoogleForCreators/web-stories-wp
# Text resizing: Revert changes to Text element from #10106

## Context



## Summary
This influences the Text element only:
- Always locks the aspect ratio if resizing a text element from corners
- Always unlocks the ratio when resizing a text element from edges.
- Resizing on the page and the aspect ratio lock in the design panel are not related 


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add a text element
2. Verify that resizing from corners is always with locked aspect ratio
3. Verify that resizing from edges is always with unlocked aspect ratio
4. Verify that toggling the aspect ratio lock in the design panel does not influence resizing on the page.


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10466
